### PR TITLE
Template project

### LINKED ISSUE #13: Package
TODO:
- [ ] Add example of using package to README @ruairimoran

### PR DESCRIPTION
**Pull request by ruairimoran**
Have a template folder that is easy to get running.

- [x] create example folder with `CMakeLists.txt` file, `src` header folder, and `main.cu` executable file
- [x] prevent example project cmake from building GPUtils targets
- [x] add readme instructions

## Associated issues
Closes #13

### CODE REVIEW

File: example/main.cu [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include <device.cuh>
```
Comment (alphaville) [inline example/main.cu]: Should this be `gputils.cuh`?

**Comment by alphaville** [comment]
In `dev` I made `Context` a singleton, so we don't need to pass it around any more (whoever wants it, can get it with `Context::getInstance()`). Perhaps you'd like to merge `dev` into `f/example` first and change the code of the example.

File: example/main.cu (hunk 1 above)
Comment (ruairimoran) [reply example/main.cu]: yes

Comment (ruairimoran) [reply example/main.cu]: had to update release

**Comment by ruairimoran** [comment]
I had to update the pre-release that cmake was fetching so that it contained header file with the new name gputils.cuh

**Review by alphaville — APPROVED** [review]
We can merge this into `dev` now if you like, but we'll have to update the example if we merge #15.

**Comment by alphaville** [comment]
**Note:** we should probably run this example in our CI.

**Pull request merged**